TESTS/mbed_platform/minimal-printf/compliance/main.cpp

```diff
@@ -856,11 +856,16 @@ static control_t test_snprintf_buffer_overflow_generic(const char *fmt, T data)
     char buffer_minimal[buf_size];
     int result_baseline;
     int result_minimal;
+#if !defined(__MICROLIB)
+    // Microlib snprintf always returns zero if the size 
+    // to copy from the buffer is zero.
+    // See reported bug https://jira.arm.com/browse/SDCOMP-54710
```

Comment:
It might be better to just leave `SDCOMP-54710` here and state there is internal microlib bug report